public/locales/en-US.json

```diff
@@ -148,9 +148,11 @@
     "header_subscriptions_cores": "Cores",
     "header_subscriptions_sockets": "Sockets",
     "header_Transfer-gibibytes": "Data transfer",
-    "measurement_Instance-hours": "{{total}} GiB hours",
-    "measurement_Storage-gibibytes": "{{total}} GiB",
-    "measurement_Transfer-gibibytes": "{{total}} GiB",
+    "header_tooltip_Storage-gibibytes": "Measurements in binary gigabyte hours",
+    "header_tooltip_Transfer-gibibytes": "Measurements in binary gigabytes",
+    "measurement_Instance-hours": "{{total}} Hours",
+    "measurement_Storage-gibibytes": "{{total}} GB hours",
```

Comment:
Is there a reason that "hours" is capitalized in one string, but not another?

Reply:
That looks like something both David and I missed Friday afternoon. So for the Instance hours column in the Current instances table, the column entries can be "X hours" (no cap).